main.yml

```diff
@@ -549,6 +549,20 @@ malware:
         title: Systems
   source_repo: https://github.com/RedHatInsights/malware-detection-frontend
 
+subscription-inventory:
+  title: Subscription Inventory
+  channel: '#subscription-inventory'
+  deployment_repo: https://github.com/RedHatInsights/subscription-inventory-ui-build
+  frontend:
```

Comment:
This configuration is no longer used. The `frontend` attribute can be entirely deleted. The only remaining thing that can be used from the `frontend` object is the `paths` attribute, but it is also no longer required:
```yaml
subscription-inventory:
  title: Subscription Inventory
  channel: '#subscription-inventory'
  deployment_repo: https://github.com/RedHatInsights/subscription-inventory-ui-build
  source_repo: https://github.com/RedHatInsights/subscription-inventory-ui
```

Reply:
Removed.